docs/development/deploying-at-scale/index.md

```diff
@@ -44,6 +44,11 @@ NOTES:
 Thanks for flying REANA 🚀
 ```
 
+!!! warning
+
+    Note that currently, REANA Helm release names are limited to 13 characters.
```

Comment:
Perhaps explain a bit more in detail, e.g. as follows:

- Note that the above `helm install` command used `reana` as the Helm release name. You can choose any other name provided that it is less than 13 characters long. (This is due to current limitation on the length of generated pod names.)

Reply:
(note also the extra blank line that CI complains about)